src/tigerbeetle/main.zig

```diff
@@ -51,7 +51,7 @@ pub fn main() !void {
     var arg_iterator = try std.process.argsWithAllocator(allocator);
     defer arg_iterator.deinit();
 
-    var command = try cli.parse_args(allocator, &arg_iterator);
+    var command = cli.parse_args(allocator, &arg_iterator);
```

Comment:
Surprisingly, this already caught something suspicious! this try was dead code, but that wasn't quite that obvious with one giant function!